StringTemplate.podspec

```diff
@@ -15,4 +15,8 @@ Pod::Spec.new do |spec|
   spec.osx.deployment_target = '10.9'
   spec.watchos.deployment_target = '2.0'
```

Comment:
These deployment targets are extremely old, to the point where Xcode no longer supports them

Reply:
https://github.com/square/StringTemplate/pull/10